src/test/java/stepDefinitions/SEDTweaksSteps.java

```diff
@@ -12,7 +26,171 @@ public SEDTweaksSteps()
 		
 	}
 	
+	@Given("^I am on a listing's details pages$")
+	public void i_am_on_a_listing_s_details_pages() throws Throwable {
+	    driver.get("https://chpl.ahrqstg.org/#/product/8481");
+	    WebDriverWait wait = new WebDriverWait(driver, 20);
+		driver.manage().window().maximize();
+		String actualTitle = driver.getTitle();
+		System.out.println("page title is : "+actualTitle);
+	    
+	}
+
+	@When("^I look at the accordions$")
+	public void i_look_at_the_accordions() throws Throwable {
+		driver.get("https://chpl.ahrqstg.org/#/product/8481");
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("div.panel:nth-child(2) > div:nth-child(1) > a:nth-child(1)")));
+		String a1 = driver.findElement(By.cssSelector("div.panel:nth-child(2) > div:nth-child(1) > a:nth-child(1)")).getText();
+	    System.out.println("Accordion title:" +a1);
+	    String a2 = driver.findElement(By.cssSelector("div.panel:nth-child(3) > div:nth-child(1) > a:nth-child(1)")).getText();
+	    System.out.println("Accordion title:" +a2);
+	    String a3 = driver.findElement(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)")).getText();
+	    System.out.println("Accordion title:" +a3);
+	    String a4 = driver.findElement(By.cssSelector("div.panel:nth-child(5) > div:nth-child(1) > a:nth-child(1)")).getText();
+	    System.out.println("Accordion title:" +a4);
+	    String a5 = driver.findElement(By.cssSelector("div.panel:nth-child(6) > div:nth-child(1) > a:nth-child(1)")).getText();
+	    System.out.println("Accordion title:" +a5);
+	    String a6 = driver.findElement(By.cssSelector("div.panel:nth-child(7) > div:nth-child(1) > a:nth-child(1)")).getText();
+	    System.out.println("Accordion title:" +a6);
+	    
+	    
+	}
+
+	@Then("^I should see them in the following order: Criteria, CQMs, SED, G(\\d+)/G(\\d+), Surveillance, Additional Info$")
```

Comment:
The "g1/g2" stuff isn't really a regex. It's literally "G1/G2", and something like "G9/G3" would be wrong